src/components/Person/Person.jsx

```diff
@@ -1 +1,33 @@
 // export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const {
+    name,
+    age,
+    sex,
+    isMarried,
+    partnerName,
+  } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
```

Comment:
No no no, refer to docs in order to fix this: https://react.dev/learn/rendering-lists